plugins/modules/data_inputs_monitors.py

```diff
@@ -0,0 +1,290 @@
+            It is therefore left out while performing idempotency checks
+        type: str
+      sourcetype:
+        description:
+          - The value to populate in the sourcetype field for incoming events.
+        type: str
+      time_before_close:
+        description:
+          - When Splunk software reaches the end of a file that is being read, the
+            file is kept open for a minimum of the number of seconds specified in
+            this value. After this period has elapsed, the file is checked again for
+            more data.
+          - This parameter is not returned back by Splunk while obtaining object information.
+            It is therefore left out while performing idempotency checks
+        type: int
+      whitelist:
+        description:
+          - Specify a regular expression for a file path. Only file paths that match this regular expression are indexed.
+        type: str
+
+  running_config:
+    description:
+    - The module, by default, will connect to the remote device and retrieve the current
+      running-config to use as a base for comparing against the contents of source.
+      There are times when it is not desirable to have the task get the current running-config
+      for every task in a playbook.  The I(running_config) argument allows the implementer
+      to pass in the configuration to use as the base config for comparison. This
+      value of this option should be the output received from device by executing
+      command.
+    type: str
+  state:
+    description:
+    - The state the configuration should be left in
+    type: str
+    choices:
+    - merged
+    - replaced
+    - deleted
+    - gathered
+    default: merged
+
+author: Pranav Bhatt (@pranav-bhatt)
+"""
+
+EXAMPLES = """
+# _________________________________________________________________
+# Using gathered
+
+- name:
+  splunk.es.data_inputs_monitors:
+    config:
+      - name: "/var/log"
+      - name: "/var"
+    state: gathered
+#
+# Output:
+#
+# "changed": false,
+# "gathered": [
+#     {
+#         "blacklist": "//var/log/[a-z0-9]/gm",
+#         "crc_salt": "<SOURCE>",
+#         "disabled": false,
+#         "host": "$decideOnStartup",
+#         "host_regex": "/(test_host)/gm",
+#         "host_segment": 3,
+#         "index": "default",
+#         "name": "/var/log",
+#         "recursive": true,
+#         "sourcetype": "test_source",
+#         "whitelist": "//var/log/[0-9]/gm"
+#     },
+#     { } # there is no configuration associated with "/var"
+# ]
+#
+# ------------------------------
+# _________________________________________________________________
+# Using merged
+- name: Example adding data input monitor with splunk.es.data_input_monitor
+  splunk.es.data_inputs_monitors:
+    config:
+      - name: "/var/log"
+        blacklist: "//var/log/[a-z]/gm"
+        check_index: True
+        check_path: True
+        crc_salt: <SOURCE>
```

Comment:
is the `<SOURCE>` actual value passed to the module?

Reply:
yes